src/vendor/README.md

```diff
@@ -0,0 +1,17 @@
+# Vendor
+
+Vendors wanting to add custom CLI commands to `otbr-agent` may implement a `ot-vendor-cli` library.
+
+The source for this library may live anywhere and does not have to be in this repository.
+
+## Usage
+
+An example implementation has been provided in [cli](cli)
+
```

Comment:
Do we want to provide any rules around naming or syntax of these commands? Long term, we want to make sure there are no collisions with commands defined by openthread.

Reply:
I don't think this is necessary. There's currently no guidance for the `otCliSetUserCommands` function https://openthread.io/reference/group/api-cli#otclisetusercommands.

It should be the dev's responsibility to make sure they aren't using any conflicting command names. Worst case, the original openthread command isn't overridden since user commands are processed only if the command isn't found in the openthread command LUT https://github.com/openthread/openthread/blob/master/src/cli/cli.cpp#L4470

Reply:
Great! Thanks for catching that cli.cpp behavior, that addresses my concerns!